config/locales/en.yml

```diff
@@ -1,2 +1,22 @@
 en:
   hello: "Hello world"
+
+  layouts:
+    application:
+      title: "Sample App"
+
+  ruby_on_rails_tut: "Ruby on Rails Tutorial"
+  rails_tut_help: "Rails Tutorial Help Page"
+  ruby_on_rails_tut_book: "Ruby on Rails Tutorial book"
+
+  static_pages:
+    home:
+      home_title: "Home" 
+      heading: "Sample App"
+      description_title: "This is the home page for the"
+      description_detail: "sample application."
+    help:
+      help_title: "Help"
+      heading: "Help"
+      description_title: "Get help on the Ruby on Rails Tutorial at the"
+      description_detail: "To get help on this sample app, please refer to"
```

Comment:
add endline for this file, please

Reply:
![image](https://user-images.githubusercontent.com/68467813/90472040-286cac80-e152-11ea-95bd-fe976e139525.png)
Done